app/views/admin/conferences/new.html.haml

```diff
@@ -6,6 +6,8 @@
                           input_html: { required: 'required' }
         = f.input :short_title, hint: "A short and unique handle for your conference, using only letters, numbers, underscores, and dashes. This will be used to identify your conference in URLs etc. Example: 'froscon2011'",
                                 input_html: { required: 'required', pattern: '[a-zA-Z0-9_-]+', title: 'Only letters, numbers, underscores, and dashes.' }, prepend: conferences_url + '/'
+        - if ENV['ORGANIZATIONS_ENABLED'] == 'true'
+          = f.input :organization, as: :select, collection: @organizations
```

Comment:
Let's just assign the default organization in the controller for now. No need to display it in the UI.